app/Http/Controllers/ChopsController.php

```diff
@@ -42,48 +64,49 @@ public function create()
      * @param  Request  $request
      * @return Response
      */
-    public function store(Request $request)
+    public function store(ChopsFormRequest $request)
     {
+
         $file = NULL;
         $shortened_url = "";
-        if(Input::hasfile('image'))
+        if($request->image)
         {
-            $upload_file = new UploadFile();
-            $file = Input::file('image');
-            $result =  $upload_file->uploadFile($file);
+
+            $file = $request->image;
+            $result =  $this->upload_file->uploadFile($file);
             if($result) {
                 $url = $result['url']; //get the url from the cloudinary result;
 
-                $shortener = new ShortenUrl();
-                $shortener->setLogin(env('BITLY_LOGIN'));
-                $shortener->setKey(env('BITLY_API_KEY'));
-                $shortener->setFormat("json");
+                $this->shortener = new ShortenUrl();
```

Comment:
You don't need to re-instantiate here because you already injected an instance of ShortenUrl from the constructor.